src/cljs/rems/navbar.cljs

```diff
@@ -34,16 +41,30 @@
                   (str/starts-with? location path))]
     [nav-link-impl path title active?]))
 
+
+(defn nav-link-options
+  "A link to path that is shown as active when the current browser location matches the path.
+
+   By default checks if path is a prefix of location, but if match-mode is :exact,
+   checks that path is exactly location."
+  [opts path title & [match-mode]]
+  (let [location @(rf/subscribe [:path])
+        active? (case match-mode
+                  :exact
+                  (= location path)
+                  ;; default: prefix
+                  (str/starts-with? location path))]
+    [nav-link-impl-opts opts path title active?]))
+
 (defn user-widget [user]
   (when user
     [:div.user-widget.px-2.px-sm-0
-     [:span
-      [:i.fa.fa-user.mr-1]
-      [:span.user-name (:name user)]]
-     [atoms/link {:id "settings" :class "nav-link"} (url-dest "/profile")
+     [nav-link-options
+      {:id "settings" :class "nav-link"}
+      (url-dest "/profile")
```

Comment:
I think `url-dest` is unnecessary here, look at the other uses of `nav-link`, they just use a relative path literally (e.g. `[nav-link "/catalogue" ...]`)

Reply:
yeah, I can see it, I did not want to refactor prematurely since I wanted to re-iterate on it anyways

Reply:
Did refactor it